tt11/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace tt1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ForegroundColor = ConsoleColor.Red;
+            Console.WriteLine("Буря");
+
+            Console.ForegroundColor = ConsoleColor.Magenta;
```

Comment:
При выводе на консоль лучше отделить название от стихотворения.
```suggestion
            Console.WriteLine("Буря");
            
            Console.WriteLine();

            Console.ForegroundColor = ConsoleColor.Magenta;
```